src/components/trade/settings/TradeSettingsRow.tsx

```diff
@@ -108,7 +106,7 @@ export const TradeSettingsRow: FC<{
           </Button>
         ))}
         <input
-          placeholder={t('common.placeholders.placeholder5')}
+          placeholder={'custom'}
```

Comment:
should this be lowercase?

Reply:
It's the same as the current value we have for it in the app. 

<img width="456" alt="image" src="https://github.com/bancorprotocol/carbon-app/assets/66181828/17df465a-5a51-4a0e-b8d3-f41234689a89">

I think it looks best as 'Custom', do you agree?